ci/rust-version.sh

```diff
@@ -38,6 +38,7 @@ export rust_nightly_docker_image=solanalabs/rust-nightly:"$nightly_version"
     declare toolchain=$1
     if ! cargo +"$toolchain" -V; then
       rustup install "$toolchain"
+      rustup default "$toolchain"
```

Comment:
We definitely don't want this line.   When a human runs something in `ci/`, their global dev environment changes on them.